esx_service/vmdk_ops.py

```diff
@@ -159,17 +159,74 @@ def RunCommand(cmd):
 
     return (s, o)
 
+
+# Translates the vmdk_path into a readable path by
+# removing the UUID string and using tenant_name
+def gets_readable_path(vmdk_path):
+    idx = 0
+    t_name = ''
+    t_uuid = ''
+    words = vmdk_path.split('/')
+    for word in words:
+        if word == DOCK_VOLS_DIR:
+            break
+        idx += 1
+
+    # The format is not always the same for vmdk_path
+    # If you suspect this is not the known format, return
+    # the original string
+    if (len(words) <= idx + 1):
+        return vmdk_path
+    t_uuid = words[idx + 1]
+    searchObj = re.search(r'[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}',
+                        t_uuid,
+                        re.M|re.I)
+    if searchObj is None:
+        return vmdk_path
+
+    # iterate over the directory and figure out
+    # which symlink's data matches the t_uuid
+    dvpath = ''
+    for i in range(1, idx + 1):
+        dvpath += '/'
```

Comment:
os.path package is your friend. Please use it for getting path components. If you really want to do it yourself, I think split("/") is an easier approach